common/urls.py

```diff
@@ -10,5 +11,6 @@
     path("success/", views.RegistrationCompleteView.as_view(), name="success"),
     path("signup/", views.SignUpView.as_view(), name="signup"),
     path("verify/", views.VerifyView.as_view(), name="verify"),
-    path("edit-profile/", login_required(views.EditProfileView.as_view()), name="edit-profile"),
+    # path("edit-profile/", login_required(views.EditProfileView.as_view()), name="edit-profile"),
```

Comment:
```suggestion
```

Reply:
And there are a couple of other commented out statements like this. Please uncomment or remove them:)

Reply:
addressed it.